public/docs/_examples/animations/ts/app/hero-list-multistep.component.ts

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
The examples are shown at different points in the guide (`upgrade.jade`) and explained in context there. Beyond that (for Plunker users basically), there's currently only the column titles in `hero-team-builder.component.ts`